ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/UI/ConnectorBindingsRhino.Receive.cs

```diff
@@ -37,14 +37,15 @@ public override async Task<StreamState> ReceiveStream(StreamState state, Progres
     converter.ReceiveMode = state.ReceiveMode;
 
     // set converter settings
+    bool settingsChanged = CurrentSettings == state.Settings ? false : true;
```

Comment:
This bool is redundant. `settingsChanged = CurrentSettings != state.settings` would do the trick

Reply:
omg haha good catch